### ISSUE TITLE
How to wire up the CI

### ISSUE THREAD

**Issue by djjudas21** [bjw-s/helm-charts#35]
Hi there. I have adopted some of the k8s-at-home charts so I can continue to maintain them, in https://github.com/k8s-at-home/charts/issues/1762

It was suggested to me to clone this repo and benefit from the CI on my own repo of charts. I've got most of it working but some of the steps seem to need a GitHub application. I haven't used one of these before, do you have any notes about how you set this up? What permissions etc.

It would be super useful to me, and probably to anyone else who wants to rehome a kah chart. Thanks

**Comment by bjw-s** [comment]
Hi @djjudas21! I am currently traveling for a bit so I can't give you an expanded answer right now, but this should be a good starting point: [GH App Docs](https://docs.github.com/en/developers/apps/building-github-apps/creating-a-github-app). You could also use a PAT, but I feel an app is generally the "cleaner" solution.

**Comment by djjudas21** [comment]
Thanks @bjw-s, I've got the app and the tokens all working now. Hopefully this is an easy question, but how does this pipeline actually work? My workflow is:

1. Make a branch
2. Edit a chart, bump `chartVersion` in `Chart.yaml`
3. Create a PR, passes checks
4. Merge PR, passes checks

The docs dir is updated at https://djjudas21.github.io/charts/docs/ but the actual repo metadata remains empty at https://djjudas21.github.io/charts/index.yaml

Is there a manual tagging step I'm missing? Helm CI stuff I've encountered before automatically generates tags/releases when it detects that `Chart.yaml` has been bumped, as in https://github.com/camerahub/charts/blob/main/.github/workflows/publish-charts.yml - no manual tagging required :slightly_smiling_face:
